CONTRIBUTING.md

```diff
@@ -1,34 +1,72 @@
-
-If you want to minimize your bug (or help minimize someone else's) for more extra credit, then you can use the [Coq bug minimizer](https://github.com/JasonGross/coq-tools) (specifically, the bug minimizer is the `find-bug.py` script in that repo).
+Bug reports are enormously useful to identify issues with Coq; we can't fix
+what we don't know about. To report a bug, please open an issue in the
+[Coq issue tracker](https://github.com/coq/coq/issues) (you'll need a GitHub
+account). You can file a bug for any of the following:
+
+- An anomaly. These are always considered bugs, so Coq will even ask you to
+  file a bug report!
+- An error you didn't expect. If you're not sure whether it's a bug or
+  intentional, feel free to file a bug anyway. We may want to improve the
+  documentation or error message.
+- Missing documentation. It's helpful to track where the documentation should
+  be improved, so please file a bug if you can't find or don't understand some
+  bit of documentation.
+- An error message that wasn't as helpful as you'd like. Bonus points for
+  suggesting what information would have helped you.
+- Bugs in CoqIDE should also be filed in the
+  [Coq issue tracker](https://github.com/coq/coq/issues).
+  Bugs in the Emacs plugin should be filed against
+  [ProofGeneral](https://github.com/ProofGeneral/PG/issues), or against
+  [company-coq](https://github.com/cpitclaudel/company-coq/issues) if they are
+  specific to company-coq features.
+
+It would help if you search the existing issues before reporting a bug. This
+can be difficult, so consider it extra credit. We don't mind duplicate bug
+reports. If unsure, you are always very welcome to ask on Discourse or Gitter
+before, after, or while writting a bug report
+
+When it applies, it's extremely helpful for bug reports to include sample
+code, and much better if the code is self-contained and complete. It's not
+necessary to minimize your bug or identify precisely where the issue is,
+since someone else can often do this if you include a complete example. We
+tend to include the code in the bug description itself, but if you have a
+very large input file then you can add it as an attachment.
+
+If you want to minimize your bug (or help minimize someone else's) for more
+extra credit, then you can use the
+[Coq bug minimizer](https://github.com/JasonGross/coq-tools) (specifically,
+the bug minimizer is the `find-bug.py` script in that repo).
+
+### Triaging bug reports
+
+Triaging bug reports (adding labels, closing outdated / resolved bugs)
+requires to be granted some permissions. You may request members of the
```

Comment:
```suggestion
requires you to be granted some permissions. You may request members of the
```

Other options:
- requires that you be granted
- requires that we grant you
- requires you to have
- requires that you have

I think the rule here is that English does not allow eliding the subject of "to be", but I'm not sure.  (I am, however, sure of the changes in this particular case.)